src/citrine/_session.py

```diff
@@ -32,11 +33,31 @@ class Session(requests.Session):
     """Wrapper around requests.Session that is both refresh-token and schema aware."""
 
     def __init__(self,
-                 refresh_token: str = environ.get('CITRINE_API_KEY'),
```

Comment:
When `environ.get` is in the method signature, the value is set at build time, which means we can't mock the value.